tests/test_scripts.py

```diff
@@ -532,7 +533,7 @@ def _create_test_migrations(self):
         for migration_dir in directories:
             if not os.path.isdir(migration_dir):
                 temp_migration_dir = tempfile.mkdtemp()
-                os.rename(temp_migration_dir, migration_dir)
+                shutil.move(temp_migration_dir, migration_dir)
```

Comment:
These tests were always failing on my machine because I have a separate `/home` partition, and `os.rename` can't move things across partitions.